pump/storage/util.go

```diff
@@ -38,6 +40,32 @@ func encodeTSKey(ts int64) []byte {
 	return buf
 }
 
+// HumanizeBytes is used for humanize configure
+type HumanizeBytes uint64
+
+// Uint64 return bytes
+func (b HumanizeBytes) Uint64() uint64 {
+	return uint64(b)
+}
+
+// UnmarshalText implements UnmarshalText
+func (b *HumanizeBytes) UnmarshalText(text []byte) error {
+	var err error
+
+	if len(text) == 0 {
+		*b = 0
+		return nil
+	}
+
+	n, err := humanize.ParseBytes(string(text))
+	if err != nil {
+		return err
```

Comment:
including `text` in the error message?

Reply:
add in https://github.com/pingcap/tidb-binlog/pull/647/commits/ac51fc0d36c528f1b232887bd9f04700792ee0b2